src/Descope.vue

```diff
@@ -55,12 +58,20 @@ defineProps({
 	},
 	errorTransformer: {
 		type: Function
+	},
+	form: {
+		type: Object
+	},
+	client: {
+		type: Object
 	}
 });
 const emit = defineEmits(['success', 'error']);
 const { projectId, baseUrl } = useOptions();
 const sdk = useDescope();
 
+const formStr = computed(() => JSON.stringify(props.form));
+const clientStr = computed(() => JSON.stringify(props.client));
```

Comment:
Did you check it when there is no value for client & form?

Reply:
i shouldn't need to, `undefined` will remain undefined and empty object will be empty

Reply:
@nirgur what do you think?

Reply:
@Bars92 why not simply check in a test ?

Reply:
I think we should parse it only if it's an object or at least have a value, otherwise I would set it to undefined or something

My concern is that we will have something like `<descope-wc form="undefined" />
And later we will try to parse it in the SDK

WDYT?